.env.heroku

```diff
@@ -0,0 +1,5 @@
+DB_USER=gpnkxlnemisllb
+DB_PASSWORD=a4f1a4ce17d87632223fdc13667d90d496d1cf5c3db80dca6b1215cd4db3c3e9
```

Comment:
It's strongly recommended to not put sensitive data here like password 😨 
Please, remove it and add it in the heroku env variables (Project > Settings > Reveal Config Vars and add the DB_PASSWORD with the value. It will be available in the dyno)